lib/impress.application.js

```diff
@@ -311,6 +311,7 @@ impress.application.mixin = (application) => {
         api.fs.stat(filePath, (err, stats) => {
           if (err || !stats.isFile()) return cb();
           application.createScript(filePath, (err, exports) => {
+            if (err) return cb();
```

Comment:
And from what I see, `callback` actually accepts an error as its first parameter.